src/Bicep.Core.UnitTests/Diagnostics/LinterRuleTests/UseSafeAccessRuleTests.cs

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Bicep.Core.Analyzers.Linter.Rules;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Bicep.Core.UnitTests.Diagnostics.LinterRuleTests;
+
+[TestClass]
+public class UseSafeAccessRuleTests : LinterRuleTestsBase
+{
+    private void AssertCodeFix(string inputFile, string resultFile)
+        => AssertCodeFix(UseSafeAccessRule.Code, "Use the safe access (.?) operator", inputFile, resultFile);
+
+    private void AssertNoDiagnostics(string inputFile)
+        => AssertLinterRuleDiagnostics(UseSafeAccessRule.Code, inputFile, [], new(OnCompileErrors.Ignore, IncludePosition.None));
+
+    [TestMethod]
+    public void Codefix_fixes_syntax_which_can_be_simplified() => AssertCodeFix("""
+param foo object
```

Comment:
nit: Since """ removes leading spaces, it would be nice to have these indented.

Reply:
My preference is not to indent, even though it is supported by C#. For some tests, the preceding whitespace is meaningful to Bicep. I also like to be able to copy+paste directly from `.bicep` file <-> code block; I find the formatting is often broken by the editor when attempting to indent.